src/diff/children.js

```diff
@@ -110,15 +114,32 @@ export function diffChildren(parentDom, newParentVNode, oldParentVNode, context,
 				}
 
 				oldDom = newDom.nextSibling;
+
+				if (firstChildDom == null) {
+					firstChildDom = newDom;
+				}
+
+				if (typeof newParentVNode.type == 'function') {
+					newParentVNode._lastDomChild = newDom;
```

Comment:
Hmm I think `_lastDomChild` has become a proxy for `typeof vnode.type == 'function' && (vnode's children have been diffed)`. Perhaps there is a better way to capture this information that is more explicit and smaller in size...

Reply:
I'm wrong. The real purpose for `_lastDomChild` is to properly resume the diff from the end of Fragment (line 92 in children.js)